### PR TITLE
:sparkles: add support for cropper v1

### PR DESCRIPTION
**Pull request by ianardee**
## Description
autogenerated

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

File: src/main/java/com/mindee/product/cropper/CropperV1Document.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CropperV1Document {
+
+
+  @Override
+  public String toString() {
+    StringBuilder outStr = new StringBuilder();
+    return SummaryHelper.cleanSummary(outStr.toString());
+  }
+}
```
Comment (sebastianMindee) [inline src/main/java/com/mindee/product/cropper/CropperV1Document.java:23]: Mmmh no.

File: docs/code_samples/cropper_v1.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    );
+
+    // Print a summary of the response
+    System.out.println(response.toString());
+
+    // Print a summary of the predictions
+//  System.out.println(response.getDocument().toString());
+
+    // Print the document-level predictions
+//    System.out.println(response.getDocument().getInference().getPrediction().toString());
```
Comment (sebastianMindee) [inline docs/code_samples/cropper_v1.txt:33]: Literally impossible :upside_down_face:

**Review by sebastianMindee — CHANGES_REQUESTED** [review]
I think some auto-generated code should be removed for empty docs.

File: src/main/java/com/mindee/product/cropper/CropperV1Document.java (hunk 1 above)
Comment (ianardee) [reply src/main/java/com/mindee/product/cropper/CropperV1Document.java]: done

File: docs/code_samples/cropper_v1.txt (hunk 2 above)
Comment (ianardee) [reply docs/code_samples/cropper_v1.txt]: removed

File: src/main/java/com/mindee/product/cropper/CropperV1Document.java (hunk 1 above)
Comment (sebastianMindee) [reply src/main/java/com/mindee/product/cropper/CropperV1Document.java]: :grin:

**Pull request merged**